src/compute/src/lib.rs

```diff
@@ -73,6 +73,11 @@ pub struct ComputeNodeOpts {
     /// Enable reporting tracing information to jaeger
     #[clap(long)]
     pub enable_jaeger_tracing: bool,
+
+    /// The number of parallel units.
+    /// We will use the number of cpu cores if not specified.
+    #[clap(long, default_value = "0")]
```

Comment:
Try this?
```suggestion
    #[clap(long, default_value_t = num_cpus::get())]
```

https://github.com/clap-rs/clap/blob/v3.2.7/examples/tutorial_derive/03_05_default_values.rs